python/main.py

```diff
@@ -113,9 +119,10 @@ def read_items():
         INNER JOIN category ON items.category_id = category.id
     ''')
     item_results = cursor.fetchall()
+    items = [{"id": item[0], "name": item[1], "category": item[2], "image_filename": item[3]} for item in item_results]
```

Comment:
is this correct?

Reply:
hello! it does work as intended at this point, but if this code is incorrect in anyway (like we’re not supposed to this or something) please let me know and i’ll do a little more research!

Reply:
The logic looks good to me 👍

Reply:
yeah, just confused with python syntax lol